task1.js

```diff
@@ -0,0 +1,11 @@
+const process = require('process');
+
+process.stdin.on('data', (data) => {
+  const reversedOutput = data.toLocaleString().split('').reverse().join('');
+  process.stdout.write(reversedOutput + '\n');
+})
+
+process.stdin.on('close', () => {
```

Comment:
`process.on('SIGINT', () => {`

Reply:
Not work too
![image](https://user-images.githubusercontent.com/21951968/208933791-ed39fa7a-e451-46ff-8c55-7352bfd1c095.png)